README.md

```diff
@@ -12,6 +12,15 @@ Setting up
 4. Go to `localhost:3000/users/<vine-user-id>/atom` or 
    `localhost:3000/users/<vine-user-id>/rss`
 
+Usage with systemd
```

Comment:
Can you add a note to item 3 above (under "setting up") to mention systemd? Maybe just ", or use systemd; see below".